openstack/compute/v2/extensions/resetstate/requests.go

```diff
@@ -0,0 +1,30 @@
+package resetstate
+
+import (
+	"errors"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+//ResetStatus refers to the states usable in ResetState Action
+
+type ResetStatus string
+
+const (
+	//ResetStateActive returns the state of the server as active
+	ResetStateActive ResetStatus = "active"
```

Comment:
How about `StateActive` and `StateError` for the const names? This flows a little better when someone is using it:

```go
err := resetstate.ResetState(client, id, resetstate.StateActive).ExtractErr()
```